integration_tests/content_item_requests_test.go

```diff
@@ -190,5 +192,30 @@ var _ = Describe("Content Item Requests", func() {
 			expectedResponse = HTTPTestResponse{Code: http.StatusOK, Body: string(expectedBody[:])}
 			assertSameResponse(actualResponse, &expectedResponse)
 		})
+
+		Context("with SUPPRESS_DRAFT_STORE_502_ERROR set to 1", func() {
+			It("returns the live content-store response when the draft content store is not running", func() {
```

Comment:
Is this strictly correct? Does is not return the live content-store response even if the draft content store is running, but its goroutine happens to complete after the live goroutine?